backend/twitterbot/tests/test_tournesolbot.py

```diff
@@ -156,41 +168,41 @@ def test_prepare_tweet(self, mock_get_twitter_account_from_video_id):
             "Aujourd'hui, je recommande 'Tournesol is great! 🌻 #tournesol' de @TournesolApp, "
             "comparée 77 fois sur #Tournesol🌻 par 28 contributeurs, critères favoris:"
             "\n- Important & actionnable\n- Stimulant & suscite la réflexion"
-            "\nyoutu.be/AAAAAAAAAAA"
+            "\ntournesol.app/entities/yt:AAAAAAAAAAA"
         )
 
         mock_get_twitter_account_from_video_id.return_value = "@TournesolApp"
 
         assert prepare_tweet(self.videos[8]) == tweet_text
 
         # Test automatic shortening of the video title to fit in the tweet
-        self.videos[8].metadata["name"] = (
-            "Tournesol is great! But this title is way to long to fit in one tweet🌻 #tournesol"
-        )
+        self.videos[8].metadata[
+            "name"
+        ] = "Tournesol is great! But this title is way to long to fit in one tweet🌻 #tournesol"
 
         tweet_text_too_long = (
-            "Aujourd'hui, je recommande 'Tournesol is great! But this title is way to long to fit "
-            "in one tweet...' de @TournesolApp, comparée 77 fois sur #Tournesol🌻 par 28 "
+            "Aujourd'hui, je recommande 'Tournesol is great! But this title is way to long to...' "
+            "de @TournesolApp, comparée 77 fois sur #Tournesol🌻 par 28 "
```

Comment:
@aidanjungo Side note: this tweet text is currently limited deliberately to 274 chars. In practice, Twitter transforms URLs into shortened "t.co" links, and as a result the [length of a URL in a Tweet is always counted as 23 characters](https://developer.twitter.com/en/docs/counting-characters). 

So after the change introduced in this PR, there are ~ 15 additional characters that could be used safely to include longer titles.

Reply:
I'm not sure I understand we we would get the 15 additional characters, the YouTube link was not treated the same way by Twitter?

Reply:
It was treated the same way.   
But the previous link (`youtu.be/AAAAAAAAAAA`) was 20 chars long, and was counted as 23.
The new one (`tournesol.app/entities/yt:AAAAAAAAAAA`) is 37 chars long, and is now "shortened" to 23 chars. So with a naive truncation, some characters are lost whereas they could still be added safely.